sdk/contentsafety/ai-content-safety-rest/README.md

```diff
@@ -162,7 +185,7 @@ const analyzeTextParameters: AnalyzeTextParameters = {
   body: {
     text: inputText,
     blocklistNames: [blocklistName],
-    breakByBlocklists: false
+    haltOnBlocklistHit: false
```

Comment:
why is this renamed ? I just want to confirm if this renamed in the payload not in the client.tsp ?

Reply:
Yes, we have breaking changes to the API contract in this version. This parameter has been renamed to "haltOnBlocklistHit".
![image](https://github.com/Azure/azure-sdk-for-js/assets/20239605/5c5c042b-ff45-401e-8dc8-8f310dc32785)